embedded-graphics/src/primitives/triangle/mod.rs

```diff
@@ -131,6 +175,16 @@ impl Triangle {
             p3: points[2].into(),
         }
     }
+
+    /// Return the [determinant](https://en.wikipedia.org/wiki/Determinant) for this triangle.
```

Comment:
This method should be renamed IMO. The returned value isn't a generic determinant but a property of the triangle that gets calculated by using a determinant. If my quick google research is right, the returned value is twice the area of the triangle.

Reply:
This is my very shallow understanding of the maths showing through. I did see it used as part of the area calculation though. How about I call it `area_doubled`? Open to better suggestions.

Reply:
> How about I call it `area_doubled`?

Sounds good.